addons/udes_routing/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "UDES Routing",
+    "summary": "udes routing",
```

Comment:
nit: UDES should probably be shouting - but i am no authority on spelling or grammar :)